### DISCUSSION TITLE
Inquiry into Mod Updating process

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [cortex-command-community/Cortex-Command-Community-Project#122] [Q&A]
Just curious what all needs to be done to make steam workshop version mods function in CCCP? there can't've been *that* many changes to how the engine works, right?

also in general there's no accessible information on this topic that isn't sealed away in a discord that frankly shouldn't be the only way to access info or get help

**Comment by VaelophisNyx** [comment]
oh wow I just literally never got an answer on this huh

**Comment by numanair** [comment]
The [mod converter](https://github.com/cortex-command-community/Cortex-Command-Legacy-Mod-Converter) does most of it. It gets [rules from the engine repo](https://github.com/MyNameIsTrez/Cortex-Command-Mod-Converter-Engine/tree/master/rules). There are probably some not in the list yet for the latest builds.
